lib/membrane/video_compositor/frame_compositor.ex

```diff
@@ -27,7 +27,7 @@ defmodule Membrane.VideoCompositor.FrameCompositor do
               internal_state :: internal_state_t,
               id :: id_t(),
               input_caps :: RawVideo.t(),
-              position :: {x :: non_neg_integer(), y :: non_neg_integer()}
+              position :: {x :: non_neg_integer(), y :: non_neg_integer(), z :: float()}
```

Comment:
1. In many 2D frameworks z-index is an integer
3. At Elixir's level of abstraction 'third dimension' has no sense, it should be a separate field